### PR TITLE
docs: mention how to register a Windows desktop with tctl

### LINKED ISSUE #31937: `tctl` resource reference is missing `windows_desktop`
## Applies To

https://goteleport.com/docs/reference/resources/

## Details

For non-AD desktop access, users must manually create the windows desktop resources. We provide an example Go implementation, but this is also possible using `tctl` since https://github.com/gravitational/teleport/pull/27192/files . This is probably simpler for most smaller customers who aren't familiar with Go.

## How will we know this is resolved?

When the `windows_desktop` resource is documented.

## Related Issues

https://github.com/gravitational/teleport/issues/9949

### PR DESCRIPTION
**Pull request by zmb3**
Fixes #31937

### CODE REVIEW

File: docs/pages/reference/resources.mdx [hunk 1]
```diff
@@ -102,6 +103,44 @@ Roles govern access to databases, SSH servers, Kubernetes clusters, web services
 
 (!docs/pages/includes/role-spec.mdx!)
 
+## Windows Desktop
```
Comment (ptgott) [inline docs/pages/reference/resources.mdx:106]:
```suggestion
## Windows desktop
```

**Review by ptgott — APPROVED** [review]

File: docs/pages/reference/resources.mdx [hunk 2]
```diff
@@ -102,6 +103,44 @@ Roles govern access to databases, SSH servers, Kubernetes clusters, web services
...
+However, you can also manage `windows_desktop` resources manually with `tctl`.
+This can be useful for managing inventories of hosts that are not joined to
+an Active Directory domain.
+
+There are a few important considerations to keep in mind when registering
+desktops this way:
+
+1. The desktop's `addr` can be a hostname or IP address, and should include
+   the RDP port (typically 3389).
+1. The desktop's `host_id` should be set to the name of a
```
Comment (strideynet) [inline docs/pages/reference/resources.mdx:120]: name or id ? I think name is a bit misleading here.

**Review by strideynet — APPROVED** [review]
Thanks for hopping on this - just one suggestion.

File: docs/pages/reference/resources.mdx (hunk 2 above)
Comment (zmb3) [reply docs/pages/reference/resources.mdx:120]: The name of the w_d_s is the host_id of the desktop.

```
➜ tctl get windows_desktop_service
kind: windows_desktop_service
metadata:
  expires: "2023-09-15T17:02:32.737179Z"
  id: 1694796752737618000
  name: 307e091b-7f6b-42e0-b78d-3362ad10b55d
spec:
  addr: 127.0.0.1:3028
  hostname: mbp16
  teleport_version: 15.0.0-dev
version: v3
```

**Pull request merged**